Framework/Algorithms/test/CalculateIqtTest.h

```diff
@@ -0,0 +1,96 @@
+#ifndef CALCULATEIQTTEST_H_
+#define CALCULATEIQTTEST_H_
+
+#include "MantidAPI/AlgorithmManager.h"
+#include "MantidAPI/FrameworkManager.h"
+#include "MantidAlgorithms/CalculateIqt.h"
+
+#include <cxxtest/TestSuite.h>
+
+#include "MantidTestHelpers/WorkspaceCreationHelper.h"
+
+using namespace Mantid::API;
+using namespace Mantid::Kernel;
+
+class CalculateIqtTest : public CxxTest::TestSuite {
+public:
+  static CalculateIqtTest *createSuite() { return new CalculateIqtTest(); }
+  static void destroySuite(CalculateIqtTest *suite) { delete suite; }
+  MatrixWorkspace_sptr m_sampleWorkspace;
```

Comment:
Is there a way to make these `const` to be absolutely sure they can't be modified between tests?

Reply:
I can make the pointer `const`, which is something. Unfortunately I don't think I  can make the underlying workspace `const` since algorithms generally don't accept `MatrixWorkspace_const_sptr`.